src/main/java/io/github/noeppi_noeppi/mods/bongo/network/BongoMessageType.java

```diff
@@ -0,0 +1,5 @@
+package io.github.noeppi_noeppi.mods.bongo.network;
+
+public enum BongoMessageType {
+	START, STOP, CREATE, NONE
```

Comment:
As far as I understand this NONE is used to update bongo without a specific reason. Wouldn't be GENERIC a better name then?